src/tests/GC/API/GC/Collect_Aggressive.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+public class AggressiveCollect
```

Comment:
as we talked about, you can use `GetGCMemoryInfo` to check the committed size. 

also I would strongly suggest that the test should actually try to run some work again (allocate some objects and let it do more GCs on its own and perhaps also trigger more GCs with the aggressive mode) after you induce a GC with the Aggressive mode so you know that it functions correctly.